ganttproject/data/resources/language/i18n.properties

```diff
@@ -1,9 +1,3 @@
-about.library = <html><body><table><tr><th>GanttProject Core</th></tr><tr><td><a href\="http\://commons.apache.org/csv/">Apache Commons CSV 0.1</a></tr><tr><td><a href\="http\://commons.apache.org/net/">Apache Commons FTP Client 3.0.1</a></tr><tr><td><a href\="http\://archive.apache.org/dist/httpcomponents/commons-httpclient/2.0/">Apache Commons HTTP Client 2.0</a></tr><tr><td><a href\="http\://commons.apache.org/logging/commons-logging-1.0.3/RELEASE-NOTES.txt">Apache Commons Logging 1.0.3</a></td></tr><tr><td><a href\="http\://archive.apache.org/dist/jakarta/slide/source/">Apache Jakarta Slide WebDav Client 2.1</a></td><td></td></tr><tr><td><a href\="http\://http\://balloontip.java.net/">Balloontip 1.2.1</a></td></tr><tr><td><a href\="http\://code.google.com/p/guava-libraries/">Google Guava libraries</a></td></tr><tr><td><a href\="http\://jcommander.org/">JCommander 1.17</a></td></tr><tr><td><a href\="http\://java.net/projects/swingx/sources/svn/show">SwingX 0.8</a></td><td></td></tr><tr><td><a href\="http\://www.jdom.org/">JDom 1.0</a></td><td></td></tr><tr><td><a href\="http\://www.jgoodies.com/freeware/looks/">JGoodies Looks 2.4</a></td><td></td></tr><tr><td><a href\="http\://developer.apple.com/library/mac/\#samplecode/AppleJavaExtensions/Introduction/Intro.html">Apple Java Extensions</a></td></tr><tr><th>MS Project import/export</th></tr><tr><td><a href\="http\://mpxj.sourceforge.net/">MPXJ 4.2</a></td></tr><tr><td><a href\="http\://poi.apache.org/">Apache POI 3.6</a></td></tr><tr><th>HTML/PDF Export</th></tr><tr><td><a href\="http\://archive.apache.org/dist/xmlgraphics/fop/source/">Apache FOP 0.20.5</a></td></tr><tr><td>Apache Avalon and Apache Batik as <a href\="http\://archive.apache.org/dist/xmlgraphics/fop/source/">FOP dependencies</a></td></tr><tr><td><a href\="http\://sourceforge.net/projects/itext/files/iText/iText5.1.3/">iText 5.1.3</a></td></tr></table></body></html>
-about.summary = <html><body><center><b>GanttProject {0}</b><br>(C) 2003-2017 GanttProject Team.<br>Licensed under General Public License (GPL) version 3</center></body></html>
```

Comment:
what need to do? I understand the word "revert", but I do not understand to what revision the revert? What exactly was wrong?